R/view_all_test_diff.R

```diff
@@ -0,0 +1,197 @@
+
+#' View Shinytest Diff
+#'
+#' @param dir Root app folder path
+#' @param sha git sha of base branch to look for
+#' @param ask Logical which allows for particular branches to be inspected
+#' @param ... Extra arguments passed to `shinytest::viewTestDiff`
+#' @param repo_dir Root repo folder path
+#' @export
+view_all_test_diff <- function(dir = "apps", sha = git_sha(dir), ask = interactive(), ..., repo_dir = file.path(dir, "..")) {
+  original_sys_call <- sys.call()
+  validate_core_pkgs()
+
+  validate_no_unexpected_shinytest_folders(dir)
```

Comment:
```suggestion
  app_dir <- validate_no_unexpected_shinytest_folders(normalizePath(dir, mustWork = TRUE))
```

Then you could remove line 35

Reply:
Line 35 is more for 

```r
  git_cmd_ <- function(..., dir = app_dir) {
    git_cmd(dir, paste0(...))
  }
```

-----------

If I have 
```r
  git_cmd_ <- function(..., dir = dir) {
    git_cmd(dir, paste0(...))
  }
```
, it does not behave well.

--------------

I guess we could change it internally to 

```r
  git_cmd_ <- function(..., git_dir = dir) {
    git_cmd(git_dir, paste0(...))
  }
```

... doing this!